german/the-black-hack.md

```diff
@@ -46,9 +46,9 @@ Rüstung bietet Schutz, indem sie allen erlittenen Schaden reduziert. Jedes Stü
 
 |     Typ        | Punkte |
 |:--------------:|:------:|
-|  Festes Wams   |   2    |
-|  Lederrüstung  |   4    |
-|   Kettenhemd   |   6    |
+| Gambeson       |   2    |
```

Comment:
Dass für einen Begriff ein Wikipediaartikel existiert oder dass es ein Wort in anderer Sprache gibt, ist für mich kein Argument, dass es sich um einen verbreiteten deutschen Begriff handelt - da ließen sich beliebig absurde Beispiele konstruieren.

Ich habe bei dem Begriff halt nicht das Gefühl, gutes Deutsch zu schreiben. Ich lasse mich da gern eines Besseren belehren (Eintrag im Wörterbuch, Verbreitung des Wortes im Internet etc.), aber "bei DSA hieß das schon immer so" oder "im Englischen gibt es das doch auch" oder "ich kenne das Wort aber" sind ja genau so subjektiv.